VideoLocker/src/main/java/org/edx/mobile/module/notification/EdxParsePushBroadcastReceiver.java

```diff
@@ -0,0 +1,38 @@
+package org.edx.mobile.module.notification;
+
+import com.parse.ParsePushBroadcastReceiver;
+
+import org.edx.mobile.view.CourseDetailTabActivity;
+import org.edx.mobile.view.MyCoursesListActivity;
+
+/**
+ * subclass ParsePushBroadcastReceiver to provide fine control of
+ * app's behavior on receiving the notification message
+ */
+public class EdxParsePushBroadcastReceiver extends ParsePushBroadcastReceiver {
+
+    public void onReceive(android.content.Context context, android.content.Intent intent) {
```

Comment:
Got it. We should probably mark any placeholders we're trying to land on master with comments. This might also be the case where even though there's only one developer we should work with a feature branch. Checking in a lot of incomplete code makes me nervous.